dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/Serializer.java

```diff
@@ -69,11 +74,16 @@ static DimensionList fromLabels(Labels labels) {
 
         if (isDelta) {
           builder.setLongCounterValueDelta(point.getValue());
+          lines.add(builder.serialize());
         } else {
-          builder.setLongCounterValueTotal(point.getValue());
+          Long delta = deltaConverter.convertLongTotalToDelta(metric.getName(), point);
+          if (delta != null) {
+            builder.setLongCounterValueDelta(delta);
+            lines.add(builder.serialize());
+          } else {
+            logger.finest(String.format(TEMPLATE_MSG_FIRST_CUMULATIVE_VALUE, metric.getName()));
```

Comment:
I assume "finest" is equivalent to trace?

Reply:
Yeah, it's basically very fine debug logging.